changelogs/fragments/32-selfhosted.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - base_url - Support for using a private instance of Healthchecks.io (https://github.com/ansible-collections/community.healthchecksio/pull/32).
```

Comment:
The first word of a changelog fragment is supposed to be lower case, i.e., "support". Also, the scope of this change is pretty much collection-wide, so, perhaps drop it altogether.

```yaml
minor_changes:
- Add support for using a private instance of Healthchecks.io (https://github.com/ansible-collections/community.healthchecksio/pull/32).
```